website/screens/components/switch/code/examples/uncontrolled.ts

```diff
@@ -1,20 +1,44 @@
-import { DxcSwitch, DxcInset } from "@dxc-technology/halstack-react";
+import {
+  DxcSwitch,
+  DxcInset,
+  DxcFlex,
+  DxcButton,
+} from "@dxc-technology/halstack-react";
+import { useRef } from "react";
 
 const code = `() => {
+  const switchRef = useRef();
+
+  const handleSubmit = () => {
+    const switchEl = switchRef.current.getElementsByTagName("input")[0];
+    console.log(switchEl.value);
```

Comment:
The same as in the Checkbox example: _"I think we can simulate the real submit event behaviour and make the example more useful by simply replacing this line with: `console.log(checkbox.checked ? checkbox.value : "");`. Maybe they find it a bit confusing that the value is still being printed even if the checkbox is unchecked."_

Reply:
Done